src/udev/cdrom_id/cdrom_id.c

```diff
@@ -124,16 +124,19 @@ static unsigned cd_media_track_count_data;
 static unsigned cd_media_track_count_audio;
 static unsigned long long int cd_media_session_last_offset;
 
-#define ERRCODE(s)        ((((s)[2] & 0x0F) << 16) | ((s)[12] << 8) | ((s)[13]))
-#define SK(errcode)        (((errcode) >> 16) & 0xF)
-#define ASC(errcode)        (((errcode) >> 8) & 0xFF)
-#define ASCQ(errcode)        ((errcode) & 0xFF)
-
-static void info_scsi_cmd_err(const char *cmd, int err) {
-        if (err == -1)
-                log_debug("%s failed", cmd);
-        else
-                log_debug("%s failed with SK=%Xh/ASC=%02Xh/ACQ=%02Xh", cmd, SK(err), ASC(err), ASCQ(err));
+#define ERRCODE(s)      ((((s)[2] & 0x0F) << 16) | ((s)[12] << 8) | ((s)[13]))
+#define SK(errcode)     (((errcode) >> 16) & 0xF)
+#define ASC(errcode)    (((errcode) >> 8) & 0xFF)
+#define ASCQ(errcode)   ((errcode) & 0xFF)
+#define CHECK_CONDITION 0x01
+
+static int log_scsi_debug_errno(int error, const char *msg) {
```

Comment:
maybe add assert:

```c
assert(error != 0);
```

(since as I understand the function doesn't make sense for error == 0...

Also, maybe add comment: `/* error < 0 means errno-style error, error > 0 means SCSI error */` or so